jupyterlab_gcedetails/src/components/resource_utilization_charts.tsx

```diff
@@ -48,6 +55,9 @@ const STYLES = stylesheet({
   utilizationChartsContainer: {
     padding: '10px 20px 20px 0px',
   },
+  flexspan: {
```

Comment:
I think you can take advantage of some of the stuff that's already been included with `typestyle` and `csstips` such as `...csstips.horizontal` if you want a horizontal flex layout.